index.html

```diff
@@ -1033,69 +1092,247 @@ <h2>Methods</h2>
-                <li>Set <var>stream</var>'s <a>[[\State]]</a> slot to
-                <code>"closing"</code>.</li>
-                <li>If <var>stream</var> has <a>finished reading</a>, set
-                <var>stream</var>'s <a>[[\State]]</a> slot to
-                <code>"closed"</code>.</li>
-                <li>If the closing procedure has not started yet, start it by sending a STREAM
-                frame with the FIN bit set.</li>
+                <li>Let <var>stream</var> be the RTCQuicWritableStream
+                object on which <code>waitForWriteBufferedAmountBelow</code> was invoked.</li>
+                <li>Let <var>p</var> be a new promise.</li>
+                <li>If <var>stream</var>'s <a>[[\Writable]]</a> slot is
+                <code>false</code>, <a>reject</a> <var>p</var> with a
+                newly created <code>InvalidStateError</code> and abort
+                these steps.</li>
+                <li>Let <var>threshold</var> be the first argument.</li>
+                <li>When <var>stream</var>'s <a>[[\WriteBufferedAmount]]]</a> slot decreases
+                from above <var>threshold</var> to less than or equal to it,
+                <a>resolve</a> <var>p</var> with <code>undefined</code>.</li>
               </ol>
+              <table class="parameters">
+                <tbody>
+                  <tr>
+                    <th>Parameter</th>
+                    <th>Type</th>
+                    <th>Nullable</th>
+                    <th>Optional</th>
+                    <th>Description</th>
+                  </tr>
+                  <tr>
+                    <td class="prmName">threshold</td>
+                    <td class="prmType"><code>unsigned long</code></td>
+                    <td class="prmNullFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmOptFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmDesc"></td>
+                  </tr>
+                </tbody>
+              </table>
               <div>
-                <em>No parameters.</em>
+                <em>Return type:</em> <code>Promise&lt;void&gt;</code>
               </div>
+            </dd>
+        </dl>
+      </section>
+      </div>
+    </section>
+    <section id="rtcquicreadablestream-interface-mixin-definition*">
+      <h3>Interface Mixin <dfn>RTCQuicReadableStream</dfn></h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+        interface mixin RTCQuicReadableStream {
+            readonly attribute boolean readable;
+            readonly attribute unsigned long readableAmount;
+            readonly attribute Promise&lt;RTCQuicStreamAbortInfo&gt; readingAborted;
+            RTCQuicStreamReadResult readInto (Uint8Array data);
+            void abortReading (RTCQuicStreamAbortInfo abortInfo);

```

Comment:
What happened to waitForReadable?

Reply:
Yikes. I don't know how that disappeared from the interface. It's definition is still there. Anyways I added it back.